posthog/models/quick_filter.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from posthog.schema import QuickFilterType
+
+from posthog.models.utils import UUIDModel
+
+
+class QuickFilter(UUIDModel):
+    TYPE_CHOICES = [(t.value, t.value) for t in QuickFilterType]
+
+    team = models.ForeignKey("posthog.Team", on_delete=models.CASCADE)
+    domain = models.CharField(max_length=100)
+    name = models.CharField(max_length=200)
+    property_path = models.CharField(max_length=500)
+    type = models.CharField(max_length=50, choices=TYPE_CHOICES, default=QuickFilterType.MANUAL_OPTIONS.value)
+    options = models.JSONField(default=list)
```

Comment:
Any reason behind the options being a nested JSON field? I suppose it's fine I just always worry about data integrity and being able to query it / iterate on the schema in future